webcompat/static/js/lib/models/comment.js

```diff
@@ -25,5 +25,45 @@ issues.Comment = Backbone.Model.extend({
       createdAt: moment(response.created_at).fromNow(),
       rawBody: response.body
     });
+    var linkHeaderRegex = /(?=.*?(link|last|next|comments|page)).*/ 
+    if (linkHeaderRegex.test(jqXHR.xhr.getAllResponseHeaders())) { 
```

Comment:
Why the regex test? [`getResponseHeader`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getResponseHeader) will return null if the specified header doesn't exist, you can just do an if statement.